worker/src/instagram.rs

```diff
@@ -117,16 +117,16 @@ pub async fn retrieve_post(user: &User, access_token: &str) -> Result<(String, S
         let post: CaptionWrapper = client.get(Url::parse(&url)?).send().await?.json().await?;
 
         for line in post.caption.split('\n').collect::<Vec<&str>>() {
-            if line.starts_with("sig:") {
-                return Ok((line[4..].to_string(), post.permalink));
+            if line.starts_with("__sig:") {
```

Comment:
Should we accept both for the time being?

Reply:
The goal of this was the prevent false positives since rather than looking at a particular post, it crawls the last (up to 10k) posts of a person on Instagram looking for the signature. This doesn't touch the Twitter/Discord flows, so if we want to avoid the false positive, we should only go with the more specific. 

If there's a different reason to always accept `sig:`, then I guess I shouldn't change it.